R/convert_data.R

```diff
@@ -328,7 +328,7 @@ make_formula <- function(x, y, short = TRUE) {
 
 
 will_make_matrix <- function(y) {
-  if (is.matrix(y) | is.vector(y) | is.factor(y))
+  if (is.matrix(y) | is.atomic(y) | is.factor(y))
```

Comment:
```suggestion
  if (is.matrix(y) | is.atomic(y))
```
same reason as earlier